msgraph/models.go

```diff
@@ -989,6 +990,11 @@ type GroupOnPremisesProvisioningError struct {
 	Value                *string   `json:"value,omitempty"`
 }
 
+type WritebackConfiguration struct {
```

Comment:
As above (will also need reordering)

```suggestion
type GroupWritebackConfiguration struct {
```